fixtures/91-span-exec-nested.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "description": "otel-cli exec basic test and stuff",
```

Comment:
oh lol, fine if this merges before you get to this but this name could be better :)

Reply:
yes, fixed.